code/datums/manufacturing.dm

```diff
@@ -2732,6 +2732,15 @@
 	time = 9 SECONDS
 	create = 1
 	category = "Medicine"
+/datum/manufacture/empty_kit
+	name = "Empty First Aid Kit"
+	item_paths = list("MET-1", "CON-1")
```

Comment:
Why would it need conductive material?

Reply:
all the other med kits require con-1 as well

Reply:
but why tho

Reply:
took the coductive material out of the empty kit but left it in the others since I have no clue where they're used and it'd probably be a balance change if I messed with that